### ISSUE TITLE
Example file config invalid

### ISSUE THREAD

**Issue by wrfly** [turbinelabs/rotor#18]
file config:

```yaml
- cluster: example-cluster-1
  instances:
    - host: 127.0.0.1
      port: 8080
    - host: 127.0.0.1
      port: 8081
- cluster: example-cluster-2
  instances:
    - host: 127.0.0.1
      port: 8083
```

docker-compose file:

```yaml
version: "2"
services:
  rotor:
    image: turbinelabs/rotor:0.18.0
    ports:
      - 50000:50000
    environment:
      - ROTOR_CONSOLE_LEVEL=debug
      - ROTOR_CMD=file
      - ROTOR_FORMAT=yaml
      - ROTOR_FILE_FILENAME=/data/routes.yml
    volumes:
      - ./data:/data
```

error log:

```txt
rotor_1  | Jul 11 10:20:12 add4810ee356 syslog-ng[11]: EOF on control channel, closing connection;
rotor_1  | *** Running /etc/rc.local...
rotor_1  | *** Booting runit daemon...
rotor_1  | *** Runit started as PID 17
rotor_1  | *** Running /usr/local/bin/rotor.sh...
rotor_1  | Jul 11 10:20:12 add4810ee356 cron[21]: (CRON) INFO (pidfile fd = 3)
rotor_1  | Jul 11 10:20:12 add4810ee356 cron[21]: (CRON) INFO (Running @reboot jobs)
rotor_1  | [info] 2018/07/11 10:20:12 No --api.key specified. Using standalone mode: Envoys will be configured to serve on port 80, for Envoy cluster "default-cluster" in zone "default-zone".
rotor_1  | [info] 2018/07/11 10:20:12 No API key specified, the API stats backend will not be configured.
rotor_1  | [info] 2018/07/11 10:20:12 watching /data/routes.yml
rotor_1  | [info] 2018/07/11 10:20:12 watching /data
rotor_1  | [debug] 2018/07/11 10:20:12 file: reload
rotor_1  | [info] 2018/07/11 10:20:12 serving xDS on [::]:50000
rotor_1  | [info] 2018/07/11 10:20:12 log streaming enabled
rotor_1  | [info] 2018/07/11 10:20:12 Stopping XDS gRPC server
rotor_1  | file: invalid character ' ' in numeric literal
rotor_1  |
rotor_1  | *** /usr/local/bin/rotor.sh exited with status 1.
rotor_1  | *** Shutting down runit daemon (PID 17)...
rotor_1  | *** Running /etc/my_init.post_shutdown.d/10_syslog-ng.shutdown...
rotor_1  | Jul 11 10:20:13 add4810ee356 syslog-ng[11]: syslog-ng shutting down; version='3.5.6'
rotor_1  | Jul 11 10:20:13 add4810ee356 syslog-ng[11]: EOF on control channel, closing connection;
rotor_1  | *** Killing all processes...
rotorandenvoy_rotor_1 exited with code 1
```

**Comment by 9len** [comment]
The doc nay be wrong, but you need to specify ROTOR_FILE_FORMAT rather that ROTOR_FORMAT as the format flag is scoped by the file command.

**Comment by wrfly** [comment]
Thanks, I didn't read it carefully.

**Comment by 9len** [comment]
We’ll fix up the doc as well. Thanks for reporting!

**Comment by 9len** [comment]
doc was fixed in 4b7524b2b2db7cb31794b18883f5078d396d4c56, thanks for the report!

**Comment by wrfly** [comment]
Really nice and helpful maintainer, and the cool glasses.
